modules/runner/src/integrationTest/java/org/apache/ignite/internal/table/ItPublicApiColocationTest.java

```diff
@@ -284,6 +284,7 @@ private static String sqlTypeName(NativeTypeSpec type) {
             case INT32:
                 return "integer";
             case INT64:
+            case NUMBER:
```

Comment:
I thin this is not correct because there is no SQL type for NativeType.NUMBER + a number is BigInteger

Reply:
@lowka did i need to change here smth ?

Reply:
There is no need for this change because NativeType::Number is part of EXCLUDED_TYPES defined by this class.